styles.css

```diff
@@ -0,0 +1,126 @@
+html {
+  height: 100%;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  height: 100%;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.h1 {
+  font-size: 2.5rem;
+  line-height: 80%;
+  font-weight: bold;
+  color: #172b4d;
+}
+
+.link {
+  color: #7f8cff;
+  font-weight: medium;
+}
+
+.container {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  height: 95%;
+  border-bottom-left-radius: 5em;
+}
+
+.content {
+  background-image: url('background.svg');
+  background-size: 100% 80%;
+  background-repeat: no-repeat;
+  background-position: center center;
+  display: flex;
+  padding: 0 1.5em;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+}
+
+ul.nav li {
+  display: inline;
+  flex-basis: 7em;
+  flex-shrink: 0;
+  text-align: center;
+  font-weight: 500;
+}
+
+.nav {
+  padding: 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.logo {
+  font-weight: bold;
+  font-size: 0.6rem;
+  color: #7f8cff;
+}
+
+.app-bar {
+  align-items: center;
+  position: fixed;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.icons {
+  display: flex;
+}
+
+.fa-brands,
+.fa-regular {
+  flex-basis: 1.5em;
+}
+
+.purple-icon {
+  color: #7f8cff;
+}
+
+@media screen and (min-width: 768px) {
+  .menu-icon {
+    display: none;
+  }
+
+  .real-content {
+    max-width: 45%;
+  }

```

Comment:
- Please align the content of this section with the `flexbox` display property according to the project requirements. Please fix this for both desktop and mobile versions 👍

Reply:
thanks for your review, using flex-basis is ok ?